Packages/com.pg649.creaturegenerator/Runtime/SkeletonGeneration/Generators/BipedInstance.cs

```diff
@@ -17,17 +17,18 @@ public class BipedInstance : ISettingsInstance
     [ObservationOrder(7)] public readonly float FeetWidth;
     [ObservationOrder(8)] public readonly float FeetLength;
 
-    [ObservationOrder(9)] public readonly List<float> TorsoThicknesses;
-    [ObservationOrder(10)] public readonly List<float> TorsoLengths;
+    [ObservationOrder(9)] public readonly int NumTorsoSegments;
```

Comment:
Doesn't quite matter yet, since we haven't got any final trained models anyway, but I would like to keep the observation order stable, because any training might depend on that order. So group fields by category, but just grab the next highest integer for the observation order, instead of changing the order of any other fields.